surefire-providers/common-junit48/src/test/java/org/apache/maven/surefire/common/junit48/FilterFactoryTest.java

```diff
@@ -933,8 +933,7 @@ public void testWithJUnitCoreFirstClassAndSingleMethod()
     @Test
     public void testShouldRunSuite()
     {
-        TestListResolver filter = new TestListResolver( "Su?te" );
-        filter = TestListResolver.optionallyWildcardFilter( filter );
+        TestListResolver filter = new TestListResolver( "*.class" );
```

Comment:
Don't change the assertions.
It makes whole PR not trusty.

Reply:
please try:
```
        TestListResolver filter = new TestListResolver( "xxxx" );
        filter = TestListResolver.optionallyWildcardFilter( filter );
....
```

Reply:
yes, it's the same, keep this change.